woocommerce/woocommerce-admin
# Add conditional copy/content and CTA to expanded task items

Fixes #6774

This PR seeks to implement expanded task item content and CTA buttons, as well as adding the content and CTAs for all of the existing tasks. Some of the tasks have conditional content based on the progress of the store's setup.

A behavioral change was made to the Tax step so that users who can turn on WCS Tax with one click don't have to click "yes please!" twice. You'll see the `auto` query param used for that.

### Accessibility




### Screenshots

![](https://user-images.githubusercontent.com/63922/120038962-0f8f0280-bfd2-11eb-92a4-a6e33355ec9f.png)

### Detailed test instructions:

- Add the `expanded` prop [here](https://github.com/woocommerce/woocommerce-admin/pull/6956#discussion_r641807943) so you see all items as expanded and can verify their copy
- Verify the copy/CTA for all scenarios in https://github.com/woocommerce/woocommerce-admin/issues/6774 - **note that the titles are not changing** even though they differ in the screenshots
- Verify when clicking "yes please" to enable Automated Tax, the Tax step doesn't require any interaction to complete
